fieldservice_stage_server_action/tests/test_fsm_stage.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class FSMRecurringCase(TransactionCase):
+    def setUp(self):
```

Comment:
Use `setUpClass(cls)` instead, as it would be more performant.

`setUp` runs before every test case
`setUpClass` runs once for all of them